local/rest_api_gcbm/app.py

```diff
@@ -350,6 +350,43 @@ def gcbm_dynamic():
     return {"status": "Run started"}, 200
 
 
+@app.route("/gcbm/getConfig", methods=["POST"])
+def getConfig():
+    """
+    Return .json for the input .tiff files
+    ---
+    tags:
+            - gcbm
+    responses:
+            200:
+    parameters:
+            - in: body
+            name: title
+            required: true
+            schema:
+                    type: string
+            description: Name of the Simulation
+            name: file_name
+            required: true
+            schema:
+                    type: string
+            description: Name of the File
+    """
+    # Default title = Simulation
+    title = request.form.get("title") or "simulation"
+    file_name = request.form.get("file_name")
+    input_dir = f"{os.getcwd()}/input/{title}/{file_name}.json"
+
+    # Check if simulation exists or not
+    if not os.path.exists(f"{input_dir}"):
+        return {"error": "Simulation does'nt exist"}, 400
```

Comment:
Change to `Simulation with name " + title + " does not exist"

Reply:
@Namyalg I'll do the above mentioned change!